<launch>
  <include file="$(find exploration)/launch/joystick_control.launch" />
  <include file="$(find bml_vrep)/launch/pose_estimation_test.launch" />

  <node name='active_pub' pkg='rostopic' type='rostopic' args='pub -r 10 /active std_msgs/Bool "data: true"' />
  <node name='marker_aggregator' pkg='exploration' type='marker_aggregator.py' args="1.0"/>

  <group ns="robot_01">
    <group ns="markers_0">
      <node name="marker_controller" pkg="exploration" type="marker_controller.py" />
      <node name="marker_planner" pkg="exploration" type="marker_planner.py" />
    </group>
  </group>

</launch>
